.github/workflows/flake8.yml

```diff
@@ -0,0 +1,24 @@
+name: flake8
```

Comment:
why a new workflow, can't we add a step to the existing linter flow here https://github.com/transferwise/pipelinewise/blob/master/.github/workflows/linter.yml

Reply:
I thought maybe we want to have separate workflow for each job. But totally agree, it's better to have it in linter.

- [ ] Move workflow into linter.yml